### PR TITLE
issue_2126 : add toString() to JobConfig

### PR DESCRIPTION
**Pull request by SapnaDerajeRadhakrishna**
Added the toString to the 'com.hazelcast.jet.config.JobConfig' class

Checklist
- [ ] Tags Set - NA 
- [ ] Milestone Set - NA
- [ ] Any breaking changes are documented -NA
- [ ] New public APIs have `@Nonnull/@Nullable` annotations -NA
- [ ] New public APIs have `@since` tags in Javadoc -NA
- [ ] For code samples, code sample main readme is updated -NA

Links to issues fixed (if any): https://github.com/hazelcast/hazelcast-jet/issues/2126
attached is the screenshot of code testing. 
issue_2126_debug_screenshot

Fixes #NNNN

List of breaking changes:

* ..

### CODE REVIEW

File: hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java [hunk 1]
```diff
@@ -1154,4 +1154,15 @@ public int hashCode() {
                 serializerConfigs, classLoaderFactory, initialSnapshotName
         );
     }
+
+	@Override
+	public String toString() {
+		return "JobConfig [name=" + name + ", processingGuarantee=" + processingGuarantee + ", snapshotIntervalMillis="
```
Comment (cangencer) [inline hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1160]: Thanks for the PR! Only one minor comment:

* should be `JobConfig{}` as that's the convention generally used

Comment (SapnaDerajeRadhakrishna) [reply hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java]: Thanks for the review. I have updated the bracket-style to '{}'. Please let me know if any other changes need to be done.

**Comment by cangencer** [comment]
verify

**Comment by hz-highfive** [comment]
The job `jet-oss-pr-builder` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder/2611/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder/2611/jet-oss-pr-builder-2611.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain 
information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
---------SUMMARY----------
--------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.0:checkstyle (default) on project hazelcast-jet-core: An error has occurred in Checkstyle report generation. Failed during checkstyle execution: There are 7 errors reported by Checkstyle 8.29 with /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/checkstyle/checkstyle.xml ruleset. -> [Help 1]
--------------------------
---------ERRORS-----------
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1158:1: File contains tab characters (this is the first instance). [FileTabCharacter]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1160: Line is longer than 123 characters (found 127). [LineLength]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1161: Line is longer than 123 characters (found 124). [LineLength]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1162: Line is longer than 123 characters (found 136). [LineLength]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1163: Line is longer than 123 characters (found 130). [LineLength]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1164: Line is longer than 123 characters (found 125). [LineLength]
--------------------------
[ERROR] /home/jenkins/jenkins_slave/workspace/jet-oss-pr-builder_2/hazelcast-jet-core/src/main/java/com/hazelcast/jet/config/JobConfig.java:1167: Line has trailing spaces. [RegexpSingleline]
--------------------------
</details>
</pre>

**Comment by cangencer** [comment]
could you fix checkstyle issues? you can verify locally using `mvn compile`

**Comment by SapnaDerajeRadhakrishna** [comment]
updated the code, by correcting the code format.
Ran the build in my local machine, below is the build status

[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:29 min
[INFO] Finished at: 2020-04-10T16:52:46-07:00
[INFO] ------------------------------------------------------------------------

**Comment by cangencer** [comment]
verify

**Pull request merged**